contrib/org/packages.el

```diff
@@ -22,6 +22,17 @@
     toc-org
     ))
 
+(when org-mapping-style
+  (push org-mapping-style org-packages))
```

Comment:
A bit too magical maybe? I would rather `worf` and `evil-org` are both in `org-packages`. You can use `:if` in `use-package` to pick which one to install, or just make the defuns conditional on the value of `org-mapping-style`. That way both packages still show up in `SPC f e h`.